src/uct/ib/base/ib_iface.c

```diff
@@ -1441,6 +1472,9 @@ UCS_CLASS_INIT_FUNC(uct_ib_iface_t, uct_iface_ops_t *tl_ops,
     self->release_desc.cb           = uct_ib_iface_release_desc;
     self->config.qp_type            = init_attr->qp_type;
     self->config.flid_enabled       = config->flid_enabled;
+
+    memset(self->config.priority_sls, UCT_IB_SL_NUM, UCT_IB_SL_NUM);
```

Comment:
why needed? can it be part of uct_ib_iface_set_priority_sls?

Reply:
Thought we would follow the same logic of setting the config to invalid values :
```c
/* initialize to invalid value */
    self->config.sl                 = UCT_IB_SL_NUM;
    self->config.reverse_sl         = UCT_IB_SL_NUM;
```